config/passport.js

```diff
@@ -1,11 +1,12 @@
+import config from './config';
+
 var mongoose = require('mongoose'),
```

Comment:
I see we have variables declared with the `var` keyword. We have also declared multiple variables in a single line. How can you improve on this?